group.go

```diff
@@ -106,15 +107,11 @@ func NewSecondaryWithContext(ctx context.Context, primary Group) (*Secondary, co
 // Go executes a function in a goroutine.
 func (s *Secondary) Go(f Func) {
 	s.self.Go(func() (func(), error) {
-		s.cond.L.Lock()
-		for s.state == open {
-			s.cond.Wait()
-		}
-		if s.state&cancel == cancel {
-			s.cond.L.Unlock()
+		s.l.Lock()
+		s.l.Unlock()
```

Comment:
`defer` ?

Reply:
For what? No statement is inside. I use the mutex for broadcast emulation. Thus, unlock is required for ping other goroutines.

Reply:
I've missed that, sorry.